extension-style/src/main/java/com/mapbox/maps/extension/style/layers/generated/ModelLayer.kt

```diff
@@ -0,0 +1,1464 @@
+  /**
+   * Set the ModelId property
+   *
+   * @param modelId value of modelId
+   */
+  override fun modelId(modelId: String) = apply {
+    val propertyValue = PropertyValue("model-id", modelId)
+    setProperty(propertyValue)
+  }
+
+  /**
+   * This is an Expression representation of "model-id".
+   *
+   * Model to render.
+   */
+  val modelIdAsExpression: Expression?
+    /**
+     * Get the ModelId property as an Expression
+     *
+     * @return String
+     */
+    get() {
+      getPropertyValue<Expression>("model-id")?.let {
+        return it
+      }
+      modelId?.let {
+        return Expression.literal(it)
+      }
+      return null
+    }
+
+  /**
+   * Set the ModelId property
+   *
+   * @param modelId value of modelId as Expression
+   */
+  override fun modelId(modelId: Expression) = apply {
+    val propertyValue = PropertyValue("model-id", modelId)
+    setProperty(propertyValue)
+  }
+
+  /**
+   * The tint color of the model layer. model-color-mix-intensity (defaults to 0) defines tint(mix) intensity - this means that, this color is not used unless model-color-mix-intensity gets value greater than 0.
+   */
+  val modelColor: String?
+    /**
+     * Get the ModelColor property
+     *
+     * @return String
+     */
+    get() {
+      modelColorAsExpression?.let {
+        return rgbaExpressionToColorString(it)
+      }
+      return null
+    }
+
+  /**
+   * Set the ModelColor property
+   *
+   * @param modelColor value of modelColor
+   */
+  override fun modelColor(modelColor: String) = apply {
+    val propertyValue = PropertyValue("model-color", modelColor)
+    setProperty(propertyValue)
+  }
+
+  /**
+   * This is an Expression representation of "model-color".
+   *
+   * The tint color of the model layer. model-color-mix-intensity (defaults to 0) defines tint(mix) intensity - this means that, this color is not used unless model-color-mix-intensity gets value greater than 0.
+   */
+  val modelColorAsExpression: Expression?
+    /**
+     * Get the ModelColor property as an Expression
+     *
+     * @return String
+     */
+    get() {
+      getPropertyValue<Expression>("model-color")?.let {
+        return it
+      }
+      return null
+    }
+
+  /**
+   * Set the ModelColor property
+   *
+   * @param modelColor value of modelColor as Expression
+   */
+  override fun modelColor(modelColor: Expression) = apply {
+    val propertyValue = PropertyValue("model-color", modelColor)
+    setProperty(propertyValue)
+  }
+
+  /**
+   * The tint color of the model layer. model-color-mix-intensity (defaults to 0) defines tint(mix) intensity - this means that, this color is not used unless model-color-mix-intensity gets value greater than 0.
+   */
+  val modelColorAsColorInt: Int?
+    /**
+     * The tint color of the model layer. model-color-mix-intensity (defaults to 0) defines tint(mix) intensity - this means that, this color is not used unless model-color-mix-intensity gets value greater than 0.
+     *
```

Comment:
same documentation above, maybe remove this one, and keep the `@return`doc.

Reply:
ticketed at https://github.com/mapbox/mapbox-maps-android-internal/issues/1156